src/components/RouteLink.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Fragment } from 'react';
+import { Link, useMatch, useResolvedPath } from 'react-router-dom';
+
+export const RouteLink = ({ children, to }) => {
+  let resolved = useResolvedPath(to);
+  // console.log('props---->', to);
+  let match = useMatch({ path: resolved.pathname, end: true });
+  // console.log('children-->', children);
+  return (
+    <Fragment>
+      <Link style={{ fontWeight: match ? 'bold' : 'normal' }} to={to}>
+        {children}
+      </Link>
+    </Fragment>
```

Comment:
Well done in choosing to create a custom component! 👏🏽  Naming is one of the challenging part of programming. I recommend instead of `bold` and `normal`, use a name that is a descriptor of what it does instead of how it looks. For example: `isActive`.  Not a must to change. Just a suggestion to think about for future.